app/bundles/PageBundle/Tests/Controller/PublicControllerTest.php

```diff
@@ -540,7 +540,7 @@ public static function provideRedirectUrls(): \Generator
 
         yield 'With click-through parameter' => [
             'https://some.test.url/asset/1:examplefilejpg?ct=parameter',
-            'https://some.test.url/asset/1:examplefilejpg?ct=dummy_click_through',
+            'https://some.test.url/asset/1:examplefilejpg?ct=parameter&ct=dummy_click_through',
```

Comment:
@biozshock can you please take a look at this? I had to update this test to pass. You added it in https://github.com/mautic/mautic/pull/15567. I'm quite unsure whether this is OK to have the `ct` param in the query twice. I guess it will work as PHP will take the second one.

Reply:
@escopecz , sure. See https://github.com/mautic/mautic/pull/15744